.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 /styleguide
 .coverage
 .DS_Store
+.prettierrc.json
```

Comment:
Taskcluster web uses https://github.com/mozilla-frontend-infra/neutrino-preset-mozilla-frontend-infra and this includes prettier. We can safely remove `.prettierrc.json` from here.

Reply:
Hi @helfi92. Did you mean to also remove .prettierrc.json from my git repo? Thanks.